features/payment/managing_payments/browsing_payments.feature

```diff
@@ -0,0 +1,20 @@
+@managing_payments
+Feature: Browsing payments
+    In order to manage all payments regardlessly of orders
+    As an Administrator
+    I want to browse all payments in the system
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has "UPS" shipping method with "$10.00" fee
+        And the store has a product "Apple"
+        And the store allows paying with "Cash on Delivery"
+        And there is an new "#00000001" order belonging to "donald@duck.com"
+        And the order has "Apple" product
```

Comment:
You could use here the step: `there is a new "#00000001" order with "Apple" product`

Reply:
Not exactly because in scenario I'm using step where I must compare email of background and of scenario step, if I have one order I may hide this email under background but problem is when we use  more than one order.